scripts/repo-health-script.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+set -e -v
+
+# Click requires this to work cause it interfaces weirdly with python 3 ASCII default
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+WORKSPACE=$PWD
+
+# If the REPORT_DATE variable is set and not an empty string parse the date to standardize it.
+if [[ ! -z $REPORT_DATE ]]; then REPORT_DATE=$(date '+%Y-%m-%d' -d "$REPORT_DATE"); fi
+
+##########################################
+# Get list of repos in given organizations
+##########################################
+
+pip-sync -q repo_tools/requirements/base.txt
+pip install -q -e repo_tools
+cd $WORKSPACE
+touch "repositories.txt"
+for ORG_NAME in ${ORG_NAMES[@]}; do
+  get_org_repo_urls "${ORG_NAME}" --url_type https --forks --add_archived --output_file "repositories.txt" --username "${GITHUB_USER_EMAIL}" --token "${GITHUB_TOKEN}" --ignore-repo "${REPOS_TO_IGNORE}"
+done
+
+############################
+# Run checks on repositories
+############################
+
+# Install checks and dashboarding script, this should also install pytest-repo-health
+pip-sync -q edx-repo-health/requirements/base.txt
+pip install -q -e edx-repo-health
+
+# data destination folder setup
+
+METADATA_FILE_DIST="${WORKSPACE}/repo-health-data/docs/checks_metadata.yaml"
+
+failed_repos=()
+
+OUTPUT_FILE_POSTFIX="_repo_health.yaml"
+
+# Git clone each repo in org and run checks on it
+input="repositories.txt"
+COUNTER=0
+while IFS= read -r line
+do
+  cd $WORKSPACE
+  if [[ "${line}" =~ ^(git@github\.com:|https://github\.com/)([a-zA-Z0-9_.-]+?)/([a-zA-Z0-9_.-]+?)\.git$ ]]; then
+      ORG_NAME="${BASH_REMATCH[2]}"
```

Comment:
Are we indenting with two spaces or four?  I don't know if we have a consistent style across the org, but we can use just one within a file.  Four spaces would be best.